### DISCUSSION TITLE
Can't enable grub-btrfs.path

### DISCUSSION THREAD

**Discussion by vuanhatran** [Antynea/grub-btrfs#280] [Q&A] [answered]
As the title says, when I run `systemctl enable grub-btrfs.path` I got the output `Unit btrfs.path does not exist`

**Comment by Schievel1** [comment, marked as answer]
What version of grub-btrfs are you running? The path-unit is removed since 4.12.

**Reply by vuanhatran** [reply]
make sense, I'm using the lastest, ty

**Reply by Schievel1** [reply]
Grub-btrfsd completely replaced the path unit, but you can still get the path unit from an earlier version and copy it in the right place yourself.

**Reply by vuanhatran** [reply]
![IMG_20230428_201527.jpg](https://user-images.githubusercontent.com/93392095/235157580-639cc1a9-ae84-4cc2-8f3a-d79e224ec25d.jpg)

I couldn't find snapshots menu in the grub boot so I checked with the systemctl status, do you have any idea what cause this

**Reply by Schievel1** [reply]
Hey, I transfered this to an issue, see https://github.com/Antynea/grub-btrfs/issues/281
